src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/CborConformanceLevel.cs

```diff
@@ -7,49 +7,55 @@
 
 namespace System.Formats.Cbor
 {
-    /// <summary>
-    ///   Defines supported conformance modes for encoding and decoding CBOR data.
-    /// </summary>
+    /// <summary>Defines supported conformance modes for encoding and decoding CBOR data.</summary>
     public enum CborConformanceMode
     {
-        /// <summary>
-        ///   Ensures that the CBOR data is well-formed, as specified in RFC7049.
-        /// </summary>
+        /// <summary>Ensures that the CBOR data is well-formed, as specified in RFC7049.</summary>
         Lax,
 
-        /// <summary>
-        ///   Ensures that the CBOR data adheres to strict mode, as specified in RFC7049 section 3.10.
-        ///   Extends lax conformance with the following requirements:
-        ///   <list type="bullet">
-        ///   <item>Maps (major type 5) must not contain duplicate keys.</item>
-        ///   <item>Simple values (major type 7) must be encoded as small a possible and exclude the reserved values 24-31.</item>
-        ///   <item>UTF-8 string encodings must be valid.</item>
-        ///   </list>
-        /// </summary>
+        /// <summary><para>Ensures that the CBOR data adheres to strict mode, as specified in RFC7049 section 3.10.</para><para>Extends lax conformance with the following requirements:</para><list type="bullet">
```

Comment:
I feel that the paragraph element negatively impact readability. Is there any chance we could omit those (at least in summary?)

Reply:
The tool currently does not do anything when it detects `<para>`.

But @sharwell from the Roslyn team actually suggested we use `<para>` whenever we have newlines.

I also agree they impact readability negatively, and people have requested us to keep readability as the main feature of our backported comments.

@sharwell would it make sense to avoid adding `<para>` to indicate newlines? What would be a disadvantage of not adding them?